docs/reference/supported-languages.njk

```diff
@@ -8,109 +8,159 @@ supportChart:
-  </ul>
-  {% if supportChart[language].rules %}
-    <h4 class="">Rule support</h4>
-    <ul>
-      <li>Core language rules: <a href="/reference/rules/?{{data.searchName}}={{data.searchTerm}}">{{thisLang.baseRules | length}}</a>
-      </li>
-      {% if true if rules
-        .counts
-        .languages[language]
-        .frameworks | keysToArr | length > 0 else 
-          false %}
-        {% for framework, val in thisLang.frameworks %}
-          <li>{{val.name | rewriteFrameworks}} rules: <a href="/reference/rules/?{{data.searchName}}={{data.searchTerm}}&q={{val.name}}">{{val.count}}</a>
-          </li>
-          {% endfor%}
-        {% endif %}
-      </ul>
-    {% endif %}
-  {% endfor %}
-
-  {% renderTemplate "liquid,md" %}
-  ---
-We are working on adding more languages in the coming months, especially PHP, C#, Go, and Python.
-
-Let us know [on Discord](https://discord.com/invite/eaHZBJUXRF) which language you'd like to see next.   
+{% renderTemplate "liquid,md" %}
+
+## Language and framework support
+
+Bearer CLI works across a variety of languages, especially:
+- Dynamically typed languages such as Ruby or JavaScript.
+- Optionally strong typed languages such as TypeScript.
+- Strong typed languages such as Java.
+
+You can access the complete list of security rules and associated vulnerabilities supported for each language by Bearer CLI in [the rules section](/reference/rules/).
+
+### Framework support
+Bearer CLI supports the majority of frameworks, requiring only core language support to perform its analysis. However, certain frameworks may require specialized rules, in that case, we mention it in the table above in the "framework" section. If you observe any gaps in support for a particular framework, please [submit an issue](https://github.com/Bearer/bearer/issues) with relevant details and examples.
+
+### Support status definition
+
+- **General Availability (GA)**: A language at the GA stage encompasses the complete feature set of Bearer, having been rigorously tested to ensure maximum accuracy.
+- **Beta**: A language in the Beta stage is equipped with the full range of Bearer's features, offering a high degree of precision. However, it may not include all the [security rules](/explanations/reports/#security-report).
+- **Alpha**: A language at the Alpha stage does not yet include the entire Bearer feature set, particularly regarding the [security report](/explanations/reports/#security-report).
+
+
+## How do we evaluate language support?
+
+The development of a robust Static Application Security Testing (SAST) tool hinges on two crucial performance metrics: recall and precision. A modern, efficient SAST solution aims is to minimize the false positive rate, thereby preventing developers from being inundated with irrelevant findings. At the same time, it should not overlook a large number of vulnerabilities, thus avoiding the propagation of a false sense of security.
```

Comment:
There's an extra `is` type (`aims is to`), and I make an optional suggestion for readability

```suggestion
The development of a robust Static Application Security Testing (SAST) tool hinges on two crucial performance metrics: recall and precision. A modern, efficient SAST solution aims to minimize the false positive rate, to avoid inundating developers with irrelevant findings. At the same time, it should not overlook relevant vulnerabilities, or it risks giving a false sense of security.
```